pkg/cmd/roachtest/tests/kv.go

```diff
@@ -97,7 +96,7 @@ func registerKV(r registry.Registry) {
 		if opts.globalMVCCRangeTombstone {
 			settings.Env = append(settings.Env, "COCKROACH_GLOBAL_MVCC_RANGE_TOMBSTONE=true")
 		}
-		c.Start(ctx, t.L(), startOpts, settings, c.Range(1, nodes))
+		c.Start(ctx, t.L(), startOpts, settings, c.CRDBNodes())
```

Comment:
Nice! There are probably a bunch more roachtests that need to be updated.

Reply:
Yeah for sure, I wanted to wait to see if my approach was fine before doing all the global refactoring 😄  I can go ahead and start working on that though as a followup PR though.